build.sbt

```diff
@@ -32,4 +34,4 @@ scalacOptions in ThisBuild ++= Seq("-unchecked", "-deprecation", "-feature", "-l
 
 enablePlugins(GatlingPlugin)
 
-lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), "gatling-imap")
+lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), s"gatling-imap-${UUID.randomUUID().toString}")
```

Comment:
I get the idea, but wouldn't that fill up the disk after a bunch of builds?

Reply:
Trying stuff....